content/blog/2024-06-19-gsoc-environment-variables.mdx

```diff
@@ -0,0 +1,62 @@
+---
+title: "GSoC'24: Supporting User-provided, Long-lived Environmental, Variables for Unikraft Builds"
+description: |
+  The goal of this project is to improve Unikraft's
+  tooling—more especially, the Kraft command-line tool—so that it can dynamically inject
+  user-supplied variables into the build system. 
```

Comment:
```suggestion
  The goal of this project is to improve Unikraft's tooling, more especially, the `kraft` command-line too, so that it can dynamically inject user-supplied variables into the build system. 
```

No need to add spacing